sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>Can you describe some scenarios so that it make sense to separate data into two features groups?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2043/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Good point. I added this above the illustration.